### DISCUSSION TITLE
Stop zoom out for easier track point edit

### DISCUSSION THREAD

**Discussion by C6competition** [osmandapp/OsmAnd#13719] [Ideas]
Assume you have build a new track roughly with some points (100 km long).
Now you want to inspect how the routing was done. You zoom into a small area at 
the beginning of the track and check, whether routing was done as you like.
If not, you tap on your track to add/delete/move the points: The menue for the track get's opened,
you select 'edit track' and now the app zooms out, until your track fit's into the screen!!!!
In consequence of what you want to do, editing a certain point in your track, you have to find 
the point again and zoom in again to do the changes! 
Up to me this zooming out is completely unneccessary, the screen should stay zoomed in 
the small area I want to edit!!!!!! 
This will save me a lot of work and the zooming in / seaching procedure on small phone screen
has a risk of mistriggering buttons as well.

**Comment by vshcherb** [comment]
Yes, there is an issue open for 4.2 to not zoom out track if you click edit

**Reply by C6competition** [reply]
Very good, thank you very much for the quick response!
